kpi/tests/test_utils.py

```diff
@@ -210,3 +211,12 @@ def test_query_parser_no_specified_field(self):
             repr(expected_q),
             repr(parse(query_string, ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS)),
         )
+
+    def test_query_parser_default_search_too_short(self):
+        # if the search query without a field is less than a specified
+        # length of characters (currently 3), then it should
+        # throw `SearchQueryTooShortException()` from `query_parser.py`
+        query_string = 'fo'
+        with self.assertRaises(SearchQueryTooShortException) as e:
+            parse(query_string, ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS)
+        self.assertTrue('Your query is too short' in str(e.exception))
```

Comment:
As per discussed in standup, let's start to use `assert condA == condB` 

Does `str(e)` return the same thing as `str(e.exception)`?

Reply:
@noliveleger that's a good point - I'll check if `str(e)` does the same thing 👍